ydb/docs/en/core/yql/reference/yql-core/udf/list/knn.md

```diff
@@ -42,62 +85,150 @@ Distance functions return small values for close vectors, while similarity funct
 {% endnote %}
 
 Similarity functions:
-* inner product `InnerProductSimilarity` (sum of products of coordinates)
-* cosine similarity `CosineSimilarity` (inner product / vector lengths)
+* inner product `InnerProductSimilarity`, it's `dot product` for us, also known as `scalar product` (sum of products of coordinates)
+* cosine similarity `CosineSimilarity` (inner product / product of vector lengths)
 
 Distance functions:
 * cosine distance `CosineDistance` (1 - cosine similarity)
+* mahattan distance `ManhattanDistance`, also known as `L1 distance` (sum of modules of coordinate difference)
```

Comment:
It may be about time to add actual formulas here. Here's the documentation on how to do it:
https://diplodoc.com/docs/en/syntax/additional#formulas